model-engine/model_engine_server/infra/services/image_cache_service.py

```diff
@@ -162,9 +166,13 @@ async def execute(self, endpoint_infra_states: Dict[str, Tuple[bool, ModelEndpoi
                 continue
 
         images_to_cache = CachedImages(cpu=[], a10=[], a100=[], t4=[])
-        for key, val in images_to_cache_priority.items():
-            images_to_cache[key] = sorted(  # type: ignore
-                val.keys(), key=lambda image: val[image], reverse=True
-            )[:IMAGES_TO_CACHE_PER_INSTANCE_TYPE]
+        try:
+            for key, val in images_to_cache_priority.items():
+                images_to_cache[key] = sorted(  # type: ignore
+                    val.keys(), key=lambda image: val[image], reverse=True
+                )[:IMAGES_TO_CACHE_PER_INSTANCE_TYPE]
+            logger.info("sorted images to cache successfully")
+        except Exception as exc:
+            logger.warning(f"sorting had an error. Error message: {exc}. Skipping sorting...")
```

Comment:
with above changes, why would end up in a case with sorting broken? seems better to keep the integrity LRU cache than to just silently accept a _random_ cache if sorting fails

Reply:
per discussion: will take out the logging and `try/except` block that catches errors here (if things are broken due to an unhandled error, they should be obviously broken so that we know they need fixing)